src/core/ui.js

```diff
@@ -24,6 +24,14 @@ styleElement.classList.add("removeOnSave");
 
 document.head.appendChild(styleElement);
 
+let body = $('body');
```

Comment:
Nit, avoid using jQuery... we are trying to remove it from the project.

Reply:
Oops!  Sorry did not know about this. Will revert back to  normal  javascript